Libraries/LLM/README.md

```diff
@@ -1,41 +0,0 @@
-#  LLM
```

Comment:
Moved to `MLXLLM` -- git is seeing it as a delete but it was just substantially changed